repairnator/docker-images/projectscanner-dockerimage/Dockerfile

```diff
@@ -0,0 +1,56 @@
+FROM maven:3.6.1-jdk-8
+
+LABEL Description="Repairnator Pipeline docker image for k8s" Vendor="Spirals" Version="0.0.0"
+LABEL maintainer="Henry Luong <tailp@kth.se>"
+
+VOLUME ["/var/log/"]
+
+COPY configure_git.sh /root/
+COPY build_scanner.sh /root/
+COPY scanner_launcher.sh /root/
+
+RUN touch /root/project_list.txt
+RUN touch /root/z3_for_linux
+RUN chmod +x /root/*.sh
+RUN echo "Europe/Paris" > /etc/timezone
+RUN apt-get update
+RUN apt-get install cloc -y
+RUN /root/configure_git.sh
+RUN /root/build_scanner.sh
+
+# Possible values: NOOP (does nothing, usual project scanner), KUBERNETES (receive/send to ActiveMQ)
+ENV PIPELINE_MODE=NOOP
+
+# Github information is provided if the repaired builds should be submitted on some repo
+# optional for something can be empty
+# Number of hours to inspect to get last builds (e.g. 4 will mean 4 hours in the past)
```

Comment:
This comment line seems to explain `ENV SCANNER_NB_HOURS=1`, which could be put just before that line of code.